<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PeryiteUI - Main Window</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
	font: 10pt &quot;FreeSans&quot;;
 	background-color: #f7f7f7;
}

QLabel {
 	background-color: #f7f7f7;
 	color: black;
  	border: 1px solid #dddddd;
}

QPushButton {
	background-color: #78e08f;
	color: white;
	border: none;
}

QPushButton:hover {
	background-color: #3c6382;
}

QGroupBox {
	border: none;
	background-color: #eaeaea;
	font-size: 16px;
	text-align: center;
}

QGroupBox:title {
	left: 16;
	top: 8;
}</string>
   </property>
   <widget class="QGroupBox" name="GeoGenGroup">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>248</y>
      <width>317</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>GeoGen</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="Geo_generateGeoGen">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>32</y>
       <width>285</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Geo_visualizeGeoGen">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>64</y>
       <width>285</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Visualize</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GeneralGroup">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>132</y>
      <width>981</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>General</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="General_outputFolderLabel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>64</y>
       <width>771</width>
       <height>25</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>No file selected</string>
     </property>
    </widget>
    <widget class="QPushButton" name="General_outputFolderSelect">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>64</y>
       <width>160</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Output Folder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="General_configFileSelect">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>32</y>
       <width>160</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Config File</string>
     </property>
    </widget>
    <widget class="QLabel" name="General_configFileLabel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>32</y>
       <width>771</width>
       <height>25</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>No file selected</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="PopGenGroup">
    <property name="geometry">
     <rect>
      <x>349</x>
      <y>248</y>
      <width>317</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>PopGen</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="Pop_generatePopGen">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>32</y>
       <width>285</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Pop_visualizePopGen">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>64</y>
       <width>285</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Visualize</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="StrideGroup">
    <property name="geometry">
     <rect>
      <x>682</x>
      <y>248</y>
      <width>316</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Stride</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="openStrideWindow">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>48</y>
       <width>285</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Stride Window</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="logoBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1013</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
	background-color: #78e08f;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGraphicsView" name="logo">
     <property name="geometry">
      <rect>
       <x>356</x>
       <y>0</y>
       <width>300</width>
       <height>120</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
